### PR TITLE
Sharepoint & Teams components

### PR DESCRIPTION
**Pull request by gordocanchola**
#### What does this PR do?
This PR adds the Teams and Sharepoint components to the TV Wallboard

#### Who is reviewing it?

@LinkMJB @shaundmorris 

@gordocanchola

### CODE REVIEW

**Review by LinkMJB — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -59,12 +62,23 @@ RUN yarn add puppeteer@10.4.0
...
+    MSGRAPH_TENANT=$MSGRAPH_TENANT \
+    MSGRAPH_TOKEN=$MSGRAPH_TOKEN \
+    MSGRAPH_CLIENTID=$MSGRAPH_CLIENTID \
+    TEAMS_TEAMID=$TEAMS_TEAMID \
+    TEAMS_CHANID=$TEAMS_CHANID \
     WESPIRE_TOKEN=$WESPIRE_TOKEN
 
 EXPOSE 3000
 ENTRYPOINT ["dumb-init", "--"]
+#CMD [ "node", "--trace-warnings",  "server.js"]
```
Comment (shaundmorris) [inline Dockerfile:83]: ❔ keeping this for debugging? Can't remember

File: client/wallboards/TVWallboard.js [hunk 2]
```diff
@@ -4,12 +4,14 @@ import SlackComponent from "../components/SlackComponent";
 import Github from "../components/Github";
 import { LeftBox, RightBox } from "../styles/WallboardStyles";
 import Grid from "@material-ui/core/Grid";
-import MediaComponent from "../components/MediaComponent";
+//import MediaComponent from "../components/MediaComponent";
```
Comment (shaundmorris) [inline client/wallboards/TVWallboard.js:7]: Do we need this MediaComponent anymore?

**Review by shaundmorris — APPROVED** [review]

File: client/wallboards/TVWallboard.js (hunk 2 above)
Comment (LinkMJB) [reply client/wallboards/TVWallboard.js:7]: The component exists still, I think either Gordo or I left it as a breadcrumb to get back to where we came from, if the Teams or Sharepoint components didn't end up working/working well. I think I might recapture this in a TV2 wallboard, preserving the existing media elements pre-teams/pre-sharepoint

File: Dockerfile (hunk 1 above)
Comment (LinkMJB) [reply Dockerfile:83]: Yeah, I left this for an easy way to turn debugging back on

**Pull request merged**